Dockerfile

```
@@ -5,31 +5,39 @@ FROM registry.access.redhat.com/ubi9/python-312 AS appbase
 
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONUNBUFFERED=1
+ENV UV_PROJECT_ENVIRONMENT=/opt/app-root \
+    UV_COMPILE_BYTECODE=1 \
+    UV_LINK_MODE=copy \
+    UV_NO_CACHE=1 \
+    UV_PYTHON_DOWNLOADS=never
+ENV PATH="/opt/app-root/bin:$PATH"
 
-COPY requirements.txt .
+COPY pyproject.toml uv.lock .
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)